plaid/api/processor.py

```diff
@@ -18,7 +18,7 @@ def stripeBankAccountTokenCreate(self, access_token, account_id):
                                     'account_id': account_id,
                                 })
 
-    def dwollaBankAccountTokenCreate(self, access_token, account_id):
+    def dwollaProcessorTokenCreate(self, access_token, account_id):
```

Comment:
I believe this change needs to be additive and not renaming  / replacing an existing function – otherwise we'll have a breaking change.

Can we leave this as is for now – and address the dwolla function naming in a separate PR?

Reply:
yeah that makes sense. I'll remove it

Reply:
@pbernasconi removed the Dwolla-related changes. Let me know if there's anything else to change